src/app.js

```diff
@@ -1,29 +1,26 @@
-//Importamos configuracion de Express
-import app from './config/express.js';
-
-//Importamos rutas
-import routes from './routes/index.route.js';
-
-import mongoose from 'mongoose';
-
+//Para usar require, hay que importar esto
 import { createRequire } from 'module';
-
 const require = createRequire(import.meta.url);
-
 require('dotenv').config();
 
+//Importamos configuracion de del servidor
+import app from './config/server.js';
+
+// BD Mongo
+import mongoose from 'mongoose';
```

Comment:
`mongoose` is a kind of ORM for `mongo`. It facilitates setting models and queries but working with standalone `mongoDB` is pretty easy. Don't mix them up, mongo is the database itself, `mongoose` is just a handy library.